tests/SprykerEcoTest/Zed/Adyen/Business/FacadeTest.php

```diff
@@ -301,4 +301,24 @@ public function testHandleCreditCard3dResponseFromAdyen(): void
             $this->assertNotEmpty($paymentAdyenOrderItem->getSpyPaymentAdyen()->getPspReference());
         }
     }
+
+    /**
+     * @return void
+     */
+    public function testHandleOnlineTransferResponseFromAdyenWithRefusedStatus(): void
+    {
+        //Arrange
```

Comment:
```suggestion
        // Arrange
```

same for similar cases